content/en/tracing/trace_collection/_index.md

```diff
@@ -27,47 +26,98 @@ To configure your application to send traces to Datadog:
 
 If you don't use the Single Step APM Instrumentation option, APM-specific configurations are required on both the Tracer and Agent to ensure that traces can be properly received from containerized, serverless, or certain other environments. Ensure you have followed instructions for both components. 
```

Comment:
We shouldn't  assume that the reader has context about Single Step APM Instrumentation.